cmd/neofs-adm/internal/modules/morph/config.go

```diff
@@ -102,12 +104,13 @@ func setConfigCmd(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("can't initialize context: %w", err)
 	}
 
-	cs, err := wCtx.Client.GetContractStateByID(1)
+	nnsHash, err := nns.InferHash(wCtx.Client)
 	if err != nil {
-		return fmt.Errorf("can't get NNS contract info: %w", err)
+		return err
```

Comment:
lets keep the context as everywhere
```suggestion
		return fmt.Errorf("can't get NNS contract hash: %w", err)
```

Reply:
Fixed!